packages/flutter/lib/src/material/tab_controller.dart

```diff
@@ -142,6 +145,10 @@ class TabController extends ChangeNotifier {
   ///
   /// When [DefaultTabController.length] is updated, this method is called to
   /// create a new [TabController] without creating a new [AnimationController].
+  ///
+  /// This instance of [TabController] must not be used anymore and has to be
+  /// disposed since [AnimationController] was injected into the new
+  /// [TabController].
```

Comment:
Is this true? May be it is up to invoker if the instance can be used or not?

Reply:
To be honest, usage of the `TabController` after the removal of `AnimationController` from it (after the call of `_copyWith`)  is impossible because we have several “bang operators” which will throw:
https://github.com/flutter/flutter/blob/0b035e026a5244215a0dffa4870783d209b78405/packages/flutter/lib/src/material/tab_controller.dart#L152
https://github.com/flutter/flutter/blob/0b035e026a5244215a0dffa4870783d209b78405/packages/flutter/lib/src/material/tab_controller.dart#L200
https://github.com/flutter/flutter/blob/0b035e026a5244215a0dffa4870783d209b78405/packages/flutter/lib/src/material/tab_controller.dart#L210
https://github.com/flutter/flutter/blob/0b035e026a5244215a0dffa4870783d209b78405/packages/flutter/lib/src/material/tab_controller.dart#L263
https://github.com/flutter/flutter/blob/0b035e026a5244215a0dffa4870783d209b78405/packages/flutter/lib/src/material/tab_controller.dart#L270